lib/src/model/method.dart

```diff
@@ -62,7 +62,9 @@ class Method extends ModelElement
       '${enclosingElement.library.dirName}/${enclosingElement.name}/$fileName';
 
   String get fullkind {
+    // A method cannot be abstract and static at the same time
```

Comment:
Nit: add a period at the end of a comment. https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences